frontend/packages/gitops-plugin/src/components/list/GitOpsTableRow.scss

```diff
@@ -0,0 +1,6 @@
+.syncStatus {
+    max-width: '100px'
+}
+.lastDeploymentTime {
+    color: var(--pf-global--palette--black-600);
+}
```

Comment:
CSS needs to follow the [BEM naming structure](http://getbem.com/naming/) + have a package prefix... ie `.gop-` or something; we don't currently have one now that GitOps is in its own package.

Reply:
I've made the changes, but haven't committed them yet.